tests/runtime/performance/test_cli_perf.py

```diff
@@ -85,18 +86,10 @@ def orq_workflow_run(ray_cluster, orq_project_dir):
     # Parse the stdout to get the workflow ID
     res = json.loads(output.stdout)
     workflow_id = res["workflow_runs"][0]["id"]
-    # Get the results to ensure the job has finished
-    _run_corq_command(
-        [
-            "get",
-            "workflow-run-results",
-            workflow_id,
-            "-d",
-            orq_project_dir,
-            "-c",
-            "local",
-        ]
-    )
+    # Wait for the workflow to finish to make sure that getting the results will succeed
+    wf = sdk.WorkflowRun.by_id(workflow_id)
+    wf.wait_until_finished()
+
```

Comment:
The purpose of these tests were to make sure the CLI didn't slow down.

Did you mean to only remove the `_run_corq_command` call instead of moving it to after the `wf.wait_until_finished()`?

Reply:
Ah, never mind. This is the fixture :)